### PR TITLE
Add GraphQL Playground config section

### PR DESCRIPTION
**Pull request by miles-crighton**
**Target**
Tutorial documentation describing how to deploy an Apollo app to AWS Lambda using the Serverless framework. [https://www.apollographql.com/docs/apollo-server/deployment/lambda/](https://www.apollographql.com/docs/apollo-server/deployment/lambda/)

**Issue**
When accessing the API endpoint generated by Serverless' deployment to AWS (/dev/graphql), the returned Playground attempts to use /graphql by default causing polling and server communications to fail. This can be manually fixed by changing the URL in the playground to use /dev/graphql however, this can lead to confusion when not fully understood by the user. (It happened to me)

Screenshot 2020-04-04 at 17 56 44

**Changes**
This commit includes a new section describing how to set up GraphQL Playground to work with the serverless deployment detailed in the guide. I've appended the sub-section to the end of the document, but maybe this information would be useful to have during the tutorial for debugging. Feel free to edit or suggest any improvements or inform whether this change is not in line with Apollo docs. 

Thanks for taking the time to check it out.

### CODE REVIEW

File: docs/source/deployment/lambda.md [hunk 1]
```diff
@@ -227,3 +227,23 @@ exports.graphqlHandler = server.createHandler({
   },
 });
 ```
+
+## Setting up GraphQL Playground
+
+By default serverless will deploy to AWS with the stage set to development, creating an API endpoint at /dev/graphql.
```
Comment (abernix) [inline docs/source/deployment/lambda.md:233]:
```suggestion
By default, `serverless` will deploy to AWS with the `stage` set to `development` resulting in an API endpoint at `/dev/graphql`.
```

File: docs/source/deployment/lambda.md [hunk 2]
```diff
@@ -227,3 +227,23 @@ exports.graphqlHandler = server.createHandler({
...
+
+To allow GraphQL Playground to correctly use the dev endpoint, add a new configuration option to the ApolloServer instance as seen below.
```
Comment (abernix) [inline docs/source/deployment/lambda.md:235]:
```suggestion
To allow GraphQL Playground to correctly use the `dev` endpoint, add a new `endpoint` configuration within the `playground` option to the `ApolloServer` instantiation options:
```

File: docs/source/deployment/lambda.md [hunk 3]
```diff
@@ -227,3 +227,23 @@ exports.graphqlHandler = server.createHandler({
...
+```js
+//graphql.js
+
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  playground: {
+      endpoint: "/dev/graphql"
+  }
+});
```
Comment (abernix) [inline docs/source/deployment/lambda.md:246]:
```suggestion
const server = new ApolloServer({
  typeDefs,
  resolvers,
  // highlight-start
  playground: {
    endpoint: "/dev/graphql"
  }
  // highlight-end
});
```

File: docs/source/deployment/lambda.md [hunk 4]
```diff
@@ -227,3 +227,23 @@ exports.graphqlHandler = server.createHandler({
...
+```
+
+For more information on configuring GraphQL Playground see the Apollo [documentation](https://www.apollographql.com/docs/apollo-server/testing/graphql-playground/).
```
Comment (abernix) [inline docs/source/deployment/lambda.md:249]:
```suggestion
For information on additional configuration options, see [GraphQL Playground](https://www.apollographql.com/docs/apollo-server/testing/graphql-playground/).
```

**Comment by miles-crighton** [comment]
Thanks a bunch for the review and changes.

I noticed in the [netlify deploy preview](https://deploy-preview-3951--apollo-server-docs.netlify.com/docs/apollo-server/deployment/lambda/) the code //highlights seem to just create empty lines to be scrolled through. Not sure what's happening there.

**Comment by abernix** [comment]
@miles-crighton You're not seeing something like this on the [link that you linked to](https://deploy-preview-3951--apollo-server-docs.netlify.com/docs/apollo-server/deployment/lambda/#setting-up-graphql-playground)?

image

**Comment by miles-crighton** [comment]
Here is what I see on Chrome + Safari: 
Screenshot 2020-04-08 at 11 29 57

**Comment by abernix** [comment]
Ah, I see what you mean now, @miles-crighton.  That would seem to be a bug in our documentation theme, not with anything we've done here, so I'm going to go ahead and merge this for now as a net win.

Also worth noting that the test failures on this PR are unrelated and are due to some tests that landed on `master` recently that are failing intermittently. cc @trevor-scheer

**Comment by abernix** [comment]
@trevorblades It appears that the introduction of `// highlight-start` and `// hightlight-end` annotations is resulting in empty lines at the end of code-fences.  To reproduce, head on over to this [deploy preview](https://deploy-preview-3951--apollo-server-docs.netlify.com/docs/apollo-server/deployment/lambda/#setting-up-graphql-playground) and scroll within the code-fence.

Screen Recording 2020-04-08 at 13 38

Is this something you're aware of or able to address?  (I'll merge this PR, but this seems like something that can be resolved outside of this.)

**Comment by miles-crighton** [comment]
@abernix @trevorblades It appears as though the additional line numbers is a bug over at Gatsby's [remark-prism](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-remark-prismjs) plugin which I've created an [issue](https://github.com/gatsbyjs/gatsby/issues/22962) for.

In the meantime, if you still wanted to use code-fence highlighting with the intended results, you can use \`\`\`js{3-7, 9} (highlights lines 3-7 and 9 indexed from the start of the code block) highlighting selection in the fence header. This requires no changes to the current `gatsby-theme-apollo`. Although I realise this is unfortunately not as self-documenting as commented directives (// highlight-start/end).

**Pull request merged**